edb/server/args.py

```diff
@@ -39,6 +39,8 @@
 
 
 BYTES_OF_MEM_PER_CONN = 100 * 1024 * 1024  # 100MiB
+TLS_CERT_FILE_NAME = "certificate.crt"
```

Comment:
I think we should name these `edbtlscert.pem` and `edbprivkey.pem` to indicate that these are edgedb's files, and `*.pem` is a more appropriate extension given the content format.